.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+##### Description of Change
+<!--
+Thank you for your Pull Request. Please provide a description above and review
+the requirements below.
+
+Contributors guide: https://github.com/zephyrproject-rtos/zephyr/blob/master/CONTRIBUTING.rst
+-->
+
+##### Checklist
+<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->
+
+- [ ] PR description included and stakeholders cc'd
+- [ ] `sanitycheck` passes
+- [ ] tests are [changed or added]
```

Comment:
I'd remove the ``[`` and ``]`` on this line since they're not on the next line.

Should you also list adding a ``Fixes: #xxxx`` tag if it addresses and existing issue, and ``Signed-off-by:`` tag?

Reply:
Signed-off-by is for commit messages, not PRs